tools/travis/travis_linux_script.sh

```diff
@@ -1,6 +1,21 @@
 #!/bin/bash
 set -e
 
+if [ "$ARCH" = "freertoslwip" ]; then
+    echo -e "\r\n==Compile multithreaded version==" && echo -en 'travis_fold:start:script.build.freertoslwip\\r'
+    export PATH="/home/travis/build/open62541/esp/xtensa-esp32-elf/bin:$PATH"
+    export IDF_PATH=/home/travis/build/open62541/esp-idf
+    cd $IDF_PATH/examples/opcua-esp32
+    cp /home/travis/build/open62541/open62541/build/open62541.c components/open62541lib
+    sed -i '/#define UA_IPV6 LWIP_IPV6/d' /home/travis/build/open62541/open62541/build/open62541.h
```

Comment:
Instead of using sed to change the `LWIP_IPV6` define, can you modify your PR to change this line:
https://github.com/open62541/open62541/blob/46270f270d7a231e7be24711a760d271c77c5842/arch/common/ua_lwip.h#L30
into something like this:
```
#ifdef LWIP_IPV6
#define UA_IPV6 LWIP_IPV6
#endif

Reply:
Macro added with https://github.com/open62541/open62541/pull/3023/commits/5f31676c03d8fe957e07f31f0cd336b460b4cec9

But needs review if its the right place to add.